test/mixins/routes.spec.js

```diff
@@ -64,4 +64,16 @@ describe('mixins/routes', () => {
       mixin.methods.userPath(1).should.equal('/users/1/edit');
     });
   });
+
+  describe('publishedFormPath', () => {
+    it('returns form overview URL when user can route', () => {
+      mixin.methods.canRoute = () => true;
+      mixin.methods.publishedFormPath(1, 'f').should.equal('/projects/1/forms/f');
+    });
+
+    it('returns submissions page URL when user can not route', () => {
+      mixin.methods.canRoute = () => false;
```

Comment:
This is a great use case for Sinon:

```js
sinon.replace(mixin.methods, 'canRoute', () => false);
```

That will automatically restore `mixin.methods.canRoute` to its original value after the test. Currently, if we ever add tests after this one, `mixin.methods.canRoute` will still be `() => false`.

No need to change anything, just wanted to put Sinon on your radar as a useful tool!